x/skyway/keeper/cosmos-originated.go

```diff
@@ -61,6 +61,22 @@ func (k Keeper) GetChainsWithTokens(ctx context.Context) []string {
 	return r
 }
 
+func (k Keeper) GetActiveChains(ctx context.Context) []string {
```

Comment:
Can we move this to the EVM keeper? Why are we creating a getter like this for business logic from a different module?

Reply:
Oh god please tell me it's not a dependency issue ...

Reply:
It's (mostly) a visibility issue. This gets called outside the keeper, and I feel we shouldn't be exposing the `EVMKeeper` within `skyway.Keeper` just because of it.
I added a wrapper and moved the logic to the evm Keeper. Let me know what you think.

Reply:
If I didn't miss another usage, this gets called during the endblocker? 

Why not just do this:

```
chains := k.EvmKeeper.GetActiveChains(ctx)
```

That way we don't have to shallow copy the same functionality on a different keeper. I think exposing the EVM keeper here instead for usage is okay.